apps/src/dom.js

```diff
@@ -37,14 +37,14 @@ exports.getTouchEventName = function(eventName) {
 };
 
 var addEvent = function(element, eventName, handler) {
-  element.addEventListener(eventName, handler, false);
-
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
```

Comment:
click has a 300ms delay on touch devices, so we listen for touchstart to have a snappier experience.